src/server/model_server.py

```diff
@@ -0,0 +1,163 @@
+from flask import Flask, request, json, make_response, send_file
+
+import os
+import sys
+src_path = os.path.join(os.path.dirname(__file__), '..')
+util_path = os.path.join(os.path.dirname(__file__), 'util')
+train_path = os.path.join(os.path.dirname(__file__), '..', 'train')
+
+sys.path.append(src_path)
+sys.path.append(util_path)
+sys.path.append(train_path)
+
+from train import get_valid_feature_groups, is_weight_output, ModelOutputType, FeatureGroups, FeatureGroup
+from util.config import getConfig
+from util.loader import parse_filters, is_valid_model, load_json, get_model_weight, get_model_group_path, get_archived_file, METADATA_FILENAME, CHECKPOINT_FOLDERNAME
+
+###############################################
+# model request 
+
+class ModelRequest():
+    def __init__(self, metrics, output_type, source='rapl', node_type=-1, weight=False, trainer_name="", filter=""):
```

Comment:
For future reference, might be a good idea to include comments and class descriptors just to make it easier for new developers to onboard

Reply:
Thanks. Added comments for each attribute.
https://github.com/sustainable-computing-io/kepler-model-server/compare/a989d7051b8056261c5df86a5594af3704517201..04aae778b8a000ff5209aa4d7008143d2262aa96